independent-publisher-connectors/Working days/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2104 @@
+                          "description": "Wages within the period",
+                          "title": "All wages"
+                        }
+                      },
+                      "description": "All working days within the period"
+                    },
+                    "weekend_days": {
+                      "type": "object",
+                      "title": "Weekend days",
+                      "description": "All weekend days within the period",
+                      "properties": {
+                        "total": {
+                          "type": "string",
+                          "title": "Total weekend days",
+                          "description": "Total weekend days within the period"
+                        },
+                        "mondays": {
+                          "type": "string",
+                          "title": "All weekend days on Mondays",
+                          "description": "Weekend days taking place on Mondays within the period"
+                        },
+                        "tuesdays": {
+                          "type": "string",
+                          "title": "All weekend days on Tuesdays",
+                          "description": "Weekend days taking place on Tuesdays within the period"
+                        },
+                        "wednesdays": {
+                          "type": "string",
+                          "title": "All weekend days on Wednesdays",
+                          "description": "Weekend days taking place on Wednesdays within the period"
+                        },
+                        "thursdays": {
+                          "type": "string",
+                          "title": "All weekend days on Thursdays",
+                          "description": "Weekend days taking place on Thursdays within the period"
+                        },
+                        "fridays": {
+                          "type": "string",
+                          "description": "Weekend days taking place on Fridays within the period",
+                          "title": "All weekend days on Fridays"
+                        },
+                        "saturdays": {
+                          "type": "string",
+                          "description": "Weekend days taking place on Saturdays within the period",
+                          "title": "All weekend days on Saturdays"
+                        },
+                        "sundays": {
+                          "type": "string",
+                          "description": "Weekend days taking place on Sundays within the period",
+                          "title": "All weekend days on Sundays"
+                        }
+                      }
+                    },
+                    "public_holidays": {
+                      "title": "Public holidays",
+                      "type": "object",
+                      "description": "public_holidays"
```

Comment:
@tposzytek descriptions should be valid sentences. Please refer to [English strings recommendations](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
Checked everywhere and fixed.